namelist.canopy

```diff
@@ -1,15 +1,13 @@
 &FILENAMES
-! file_vars    = 'input_variables_point.txt'
-!  file_vars    = 'input_variables.txt'
-!  file_vars    = 'input_variables_1D.nc'
-  file_vars    = 'input_variables_2D.nc'
-  file_out     = 'test'
+ file_vars    = 'gfs.t12z.20220701.sfcf000.txt'
+!  file_vars    = 'gfs.t12z.20220701.sfcf000.nc'
+  file_out     = 'test_out'
```

Comment:
nc input not the default anymore? will have to adjust the CI I think

Reply:
@zmoon   The CI fails when it is NetCDF default input here.  This is because it seems the CI also checks when NETCDF=0 compilation, and if the default NL has .ncf as the input, the run will fail (with NETCDF=0).    That is why I change it to .txt as the default.

Reply:
well I'll see if I can fix it. we want to be testing both cases.

Reply:
2c31c0a

Reply:
@zmoon I assume this fixes the issue for CI to test both!

Reply:
Yes, and I added some additional check to make sure that the nml default is to run with nc input.